client/src/components/rooms/RoomsMeetingExperience.tsx

```diff
@@ -9,31 +9,44 @@ import { AzureCommunicationTokenCredential } from '@azure/communication-common';
 import { Theme, PartialTheme, Spinner } from '@fluentui/react';
 import { fullSizeStyles } from '../../styles/Common.styles';
 import { RoomParticipantRole, RoomsInfo } from '../../models/RoomModel';
+import { PostCallConfig } from '../../models/ConfigModel';
+import { Survey } from '../postcall/Survey';
 import MobileDetect from 'mobile-detect';
 
 export interface RoomsMeetingExperienceProps {
   roomsInfo: RoomsInfo;
   token: string;
+  postCall?: PostCallConfig;
   fluentTheme?: PartialTheme | Theme;
   onDisplayError(error: any): void;
 }
 
 export const RoomsMeetingExperience = (props: RoomsMeetingExperienceProps): JSX.Element => {
-  const { roomsInfo, token, fluentTheme, onDisplayError } = props;
+  const { roomsInfo, token, postCall, fluentTheme, onDisplayError } = props;
   const { userId, userRole, locator } = roomsInfo;
 
   const displayName =
     userRole === RoomParticipantRole.presenter ? 'Virtual appointments Host' : 'Virtual appointments User';
 
   const [callAdapter, setCallAdapter] = useState<CallAdapter | undefined>(undefined);
-
+  const [renderPostCall, setRenderPostCall] = useState<boolean>(false);
+  const [callId, setCallId] = useState<string>();
   const credential = useMemo(() => new AzureCommunicationTokenCredential(token), [token]);
 
   useEffect(() => {
     const _createAdapters = async (): Promise<void> => {
       try {
         const adapter = await _createCustomAdapter({ communicationUserId: userId }, credential, displayName, locator);
-
+        if (userRole === RoomParticipantRole.attendee && postCall?.survey.type) {
```

Comment:
is it better to add userRole != presenter so that we can show post call to all other roles (attendee and consumer) just in case

Reply:
Done!